src/sentry/ingest/userreport.py

```diff
@@ -72,11 +72,13 @@ def save_userreport(project, report, start_time=None):
         report_instance = existing_report
 
     else:
-        if report_instance.group:
+        if report_instance.group_id:
             report_instance.notify()
 
     user_feedback_received.send(
-        project=report_instance.project, group=report_instance.group, sender=save_userreport
+        project=Project.objects.get(id=report_instance.project_id),
```

Comment:
Please do not access the cache here (or, worse, risk a database query). Can you pass `project` and `group` in from the caller?

Reply:
Is it safe to use `event.group` here?

Reply:
Nm, this signal doesn't even define `group` and it's unused in the one receiver connected to it.